open-api.yaml

```diff
@@ -681,6 +681,22 @@ components:
           default:
             - ''
         example: 'title,description'
+    highlightPreTag:
+      name: highlightPreTag
+      in: query
+      required: false
+      description: Specify the tag to put before the highlighted query terms.
+      schema:
+        type: string
+        default: '<em>'
```

Comment:
@brunoocasali We did not change this value because it's already like that and the users seem to be used to it. Did'nt saw users reporting direct feedback about the default value.

Changing it could change all the frontend code that is already using `em` and could have applied CSS rules to it. As the settings now allow to customize this, it seemed to me to have little impact.

Reply:
Yeah, I can see your point thanks for it! While researching it, I saw Elasticsearch and Solr use the same element (`<em>`) as the default one.

Even if I disagree, I'm sold regarding preventing an expectation "breaking" for the current users, so let's keep it!

- https://solr.apache.org/guide/8_11/highlighting.html
- https://www.elastic.co/guide/en/elasticsearch/reference/current/highlighting.html